server/src/server.js

```diff
@@ -747,7 +747,7 @@ app.get("/data/:id/:domain", function(req, res) {
   }
   const shotId = `${req.params.id}/${req.params.domain}`;
   // FIXME: maybe we should allow for accountId here too:
-  Shot.getRawValue(shotId, req.deviceId).then((data) => {
+  Shot.getRawValue(shotId).then((data) => {
```

Comment:
Updating getRawValue to remove deviceId, is that ok?

Reply:
How can you tell if the user owns the shot, if you aren't passing in the `deviceId` or the `accountId`?

Reply:
Hmm. At the top of this function (line 744) is "if not req.deviceId, return 404". I think that's not correct for the case where you are logging in from a non-Firefox browser, and have `accountId` but not `deviceId`, but I need to double-check where this is used.

At this particular line, I suspect we actually need to pass in `accountId` along with `deviceId` here, if you want to be able to call this from a non-firefox browser.

Reply:
This was the only place outside of Shot.get where we are using getRawValue , so wan't sure. I will update to bring check for deviceId and accountId inside getRawValue